src/ray/object_manager/plasma/store.cc

```diff
@@ -113,9 +113,8 @@ PlasmaStore::PlasmaStore(instrumented_io_context &main_service,
                 this->AddToClientObjectIds(object_id, request->client);
               },
           [this](const auto &request) { this->ReturnFromGet(request); }) {
-  const auto event_stats_print_interval_ms =
-      RayConfig::instance().event_stats_print_interval_ms();
-  if (event_stats_print_interval_ms > 0 && RayConfig::instance().event_stats()) {
+  if (RayConfig::instance().event_stats_print_interval_ms() > 0 &&
```

Comment:
Reverted the change to have metrics recording still part of the PrintStats loop. 
So the default `event_stats_print_interval_ms` is 1min, I wonder if we should change the default? In the events `event_stats_print_interval_ms` greater than the metric_reports interval, no metrics would be there. 

If not - maybe we could have another interval?

Reply:
I actually feel we will eventually make a larger change to follow how metrics are exported just like the task metrics, i.e. use something like a counter map and register callback on the change, rather than periodically recording them from member fields. 

This is probably the better way when we add another label, e.g. job id. (Stephanie has requested such a breakdown as multiple jobs might be run at the same time).